openshift/4.0/templates/jenkins-slaves/README.md

```diff
@@ -1,5 +1,3 @@
 PIMS Jenkins pipelines requires 3 Jenkins agents to execute.
 
-1. [NPM](./jenkins-slave-npm/README.md) - for building the frontend application
```

Comment:
As far as I'm aware we have removed the usage of these agents (favoring github actions) and have no intention of re-introducing them